disperser/common/blobstore/shared_storage.go

```diff
@@ -90,6 +92,12 @@ func (s *SharedBlobStore) StoreBlob(ctx context.Context, blob *core.Blob, reques
 	metadataKey.BlobHash = blobHash
 	metadataKey.MetadataHash = metadataHash
 
+	refreshedMetadata, err := s.GetBlobMetadata(ctx, metadataKey)
```

Comment:
what if there is a 500 error from dynamo?

Reply:
I think I should just check if refreshedMetadata is nil or not